.github/workflows/main.yml

```diff
@@ -62,9 +62,44 @@ jobs:
           submodules: recursive
 
       - name: Go test
-        run: go test -coverprofile coverage.out -timeout 28m ./...
+        run: go test -shuffle=on -coverprofile coverage.out -timeout 28m ./...
 
       - name: Upload coverage file to Codecov
         uses: codecov/codecov-action@v3
         with:
           files: coverage.out
+
+  mutating:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.17.x
```

Comment:
nitpick: Should be 1.18 already

Reply:
well https://github.com/0xPolygon/go-ibft/blob/main/go.mod#L3 off by a few hours haha